### PR TITLE
Update Technical Writing team handbook

### PR DESCRIPTION
**Pull request by justinegeffen**
Rewriting sections for clarity (it's unclear who the audience is and what some of the processes refer to) and added prioritization section.

To do:
- Audience assessment (community? staff? internal?)
- Structure

### CODE REVIEW

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 1]
```diff
@@ -46,34 +46,36 @@ Other areas of documentation include:
 
 Please reach out to the [Documentation Working Group](https://community.mattermost.com/private-core/channels/dwg-documentation-working-group) for any questions.
 
-### Contributing to documentation
+## How to contribute to the documentation
 
-The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
+Our documentation is open to contributions from anyone. The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md:51]: I wonder if we want to make this even more inclusive by including details on using the Edit on GitHub option (which auto-creates a fork if the user doesn't have appropriate permissions)?

**Comment by cwarnermm** [comment]
I agree with you that we need to identify the target audience for this information, then tailor the details based on what that audience needs.

**Review by cwarnermm — DISMISSED** [review]
Good updates!

File: operations/research-and-development/product/technical-writing-team-handbook/README.md (hunk 1 above)
Comment (justinegeffen) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md:51]: The link in the next sentence does provide that information with more context but we can definitely add it here as well.

**Pull request merged**